src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -634,102 +628,137 @@ public void testIndicateMissingResponses() {
         assertFalse(resultsPage.verifyMissingResponsesVisibility());
     }
 
-    // TODO unnecessary coupling of FRComments test here. this should be tested separately.
-    private void testFeedbackResponseCommentActions() throws Exception {
```

Comment:
Why the previous test get deleted?

Reply:
I refactored tests. This function contained tests for Questions view and RGQ view. Separated these. Refactored questions view tests further so as to accommodate GQR view tests.

Reply:
But where do you separate to `displayByQuestion `?

Reply:
ok, I see it.